src/pages/seungkyun/Main/Main.js

```diff
@@ -1,8 +1,93 @@
+/*eslint-disable*/
 import React from 'react';
+import Feed from './Feed/Feed';
+import './Main.scss';
 
 class Main extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      feedList: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/feedData.json')
```

Comment:
`http://localhost:3000`은 생략할 수 있습니다.
```suggestion
    fetch('/data/feedData.json')
```